runtime/cert/example_cert.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Please add `_test` to this file, we don't want these certs in flux binary or the controllers.

Reply:
It may be even better to simply use the fixtures from https://cs.opensource.google/go/x/crypto/+/master:ssh/testdata/keys.go

Reply:
Ok I will remove the certs from this PR.

Reply:
@hiddeco I had a look at the testdata and I am a bit confused about how that can be used to replace the test certificate currently in the PR.